src/asmcnc/skavaUI/widget_quick_commands.py

```diff
@@ -157,6 +157,9 @@ def proceed_to_go_screen(self):
         # Machine must be homed.
         # Job must be within machine bounds.
 
+        self.m.spindle_health_check_failed = False
+        self.m.spindle_health_check_passed = False

```

Comment:
These need to be set somewhere else, as the go screen is also used by shapecutter and therefore spindle health check will be active from shapecutter